wp-post-meta-revisions.php

```diff
@@ -134,6 +134,10 @@ public function _wp_save_revisioned_meta_fields( $revision_id ) {
 		foreach ( $this->_wp_post_revision_meta_keys() as $meta_key ) {
 			$meta_value = get_post_meta( $post_id, $meta_key );
 
+			if ( empty( $meta_value ) ) {
```

Comment:
your a little early on the review!, was just putting this up and pondering while i made lunch, wouldn't this always be an empty array if nothing is found because `single` is false on `get_post_meta` any other values should pass, I'm only trying catching empty arrays which no meta key was found?